mw-webapp/src/component/tag/Tag.module.scss

```diff
@@ -1,7 +1,7 @@
 @import "src/styles/variables";
 
-$WayTagPadding: 10px;
-$WayTagBorderRadius: 1000px;
+$TagPadding: 10px;
+$TagBorderRadius: 1000px;
```

Comment:
Please check is it possible to use global var - tertiaryBorderRadius

Reply:
It looks completely different:
![image](https://github.com/tritonJS826/masters-way/assets/8752900/2ea95977-9e59-4d48-9123-f375a411a33c)